docs/cloudevent.js.html

```diff
@@ -0,0 +1,348 @@
+   * @returns {string} the event source
+   */
+  getSource() {
+    return this.spec.getSource();
+  }
+
+  /**
+   * Sets the event id. Source + id must be unique for each distinct event.
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#id
+   * @param {string} id source+id must be unique for each distinct event
+   * @returns {CloudEvent} this CloudEvent instance
+   */
+  id(id) {
+    this.spec.id(id);
+    return this;
+  }
+
+  /**
+   * Gets the event id.
+   * @returns {string} the event id
+   */
+  getId() {
+    return this.spec.getId();
+  }
+
+  /**
+   * Sets the timestamp for this event
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#time
+   * @param {Date} time timestamp when the event occurred
+   * @returns {CloudEvent} this CloudEvent instance
+   */
+  time(time) {
+    // TODO: Ensure that this is represented as a Date internally,
+    // or update the JSDoc
+    this.spec.time(time);
+    return this;
+  }
+
+  /**
+   * Gets the timestamp for this event
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#time
+   * @returns {Date} the timestamp for this event
+   */
+  getTime() {
+    // TODO: Ensure that this is represented as a Date internally,
+    // or update the JSDoc
+    return this.spec.getTime();
+  }
+
+  // TODO: Deprecated in 1.0
+  schemaurl(schemaurl) {
+    this.spec.schemaurl(schemaurl);
+    return this;
+  }
+
+  // TODO: Deprecated in 1.0
+  getSchemaurl() {
+    return this.spec.getSchemaurl();
+  }
+
+  /**
+   * Sets the content type of the data value for this event
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#datacontenttype
+   * @param {string} contenttype per https://tools.ietf.org/html/rfc2046
+   * @returns {CloudEvent} this CloudEvent instance
+   */
+  dataContenttype(contenttype) {
+    this.spec.dataContenttype(contenttype);
+    return this;
+  }
+
+  /**
+   * Gets the content type of the data value for this event
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#datacontenttype
+   * @returns {string} the content type for the data in this event
+   */
+  getDataContenttype() {
+    return this.spec.getDataContenttype();
+  }
+
+  /**
+   * Sets the data for this event
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#event-data
+   * @param {*} data any data associated with this event
+   * @returns {CloudEvent} this CloudEvent instance
+   */
+  data(data) {
+    this.spec.data(data);
+    return this;
+  }
+
+  /**
+   * Gets any data that has been set for this event
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#event-data
+   * @returns {*} any data set for this event
+   */
+  getData() {
+    return this.spec.getData();
+  }
+
+  /**
+   * Adds an extension attribute to this CloudEvent
+   * @see https://github.com/cloudevents/spec/blob/master/spec.md#extension-context-attributes
+   * @param {*} key the name of the exteneion attribute
```

Comment:
@lance seems the typo was fixed in code but the generated docs needs to be updated again ?

Reply:
not a blocker since it will be updated along time anyway :+1:

Reply:
Oh shoot - thanks. I'll update before landing this.